### ISSUE TITLE
Support for response body

### ISSUE THREAD

**Issue by n2o1988** [firefox-devtools/bidi-har-export#22]
Hi @juliandescottes, first of all thank you so much for this tool! 
I was wondering if there are plans to include response bodies in the generated `har` files? Not sure if that's something that's even supported by the BiDi standard today but I thought I'd ask.

I've been following developments on https://bugzilla.mozilla.org/show_bug.cgi?id=1744483 and I'm in a similar position as the user who opened the bug, where I need to get access to Network logs as part of a selenium-based automation test.

**Comment by juliandescottes** [comment]
Hi, thanks for reaching out!

At the moment response bodies are not in the BiDi spec but that might change in the future.
There's interest to expose response bodies for network interception scenarios, see https://github.com/w3c/webdriver-bidi/issues/541 . When this happens, there should at least be an option to add response bodies to network events (and then we can add it here).

**Comment by n2o1988** [comment]
Hi Julian, thanks a million for the response! I'll look forward to the specs to incorporate that option in the future 👌

**Comment by bgrins** [comment]
This would be nice. I see https://github.com/w3c/webdriver-bidi/issues/747 as well.

**Comment by soulgalore** [comment]
Hi, I forgot that we lost that information when we switched from the HAR trigger in Browsertime. Is there a way to actually get the response today through Bidi? I've been heavily using that functionality (but for Chrome) for Wikipedia. It's super useful to have the actual HTML/JavaScript from the time the test were running so I can see exact what code we had at that time.

**Comment by juliandescottes** [comment]
At the moment we still don't support getting the network response body from BiDi.

**Comment by Vinodh7887** [comment]
is there any update when we should be able to extract network response body using BiDi?

**Comment by whimboo** [comment]
Please see https://github.com/w3c/webdriver-bidi/issues/747 as reference. Hopefully in Q1 2025 we should have it included in the specification to get started.

**Comment by juliandescottes** [comment]
> Please check if this solves your problem: [1971780 - Implement the network.getData command](https://bugzilla.mozilla.org/show_bug.cgi?id=1971780) [1988955 - Add support for dataType "request" to addDataCollector, getData and disownData](https://bugzilla.mozilla.org/show_bug.cgi?id=1988955)
> 
> More Information: [w3c/webdriver-bidi#877](https://github.com/w3c/webdriver-bidi/pull/877) https://www.w3.org/TR/webdriver-bidi/#network-data-collection

Yes normally all blockers have been resolved now, so we could include response bodies now.

**Comment by juliandescottes** [comment]
Sure let's discuss it for m19.
